.clang-format

```diff
@@ -0,0 +1,34 @@
+---
+BasedOnStyle: Google
+---
+Language: Cpp
+ColumnLimit: 100
+IndentWidth: 8
+ContinuationIndentWidth: 8
+ConstructorInitializerIndentWidth: 8
+TabWidth: 8
+UseTab: ForIndentation
+MaxEmptyLinesToKeep: 3
+AlignTrailingComments: true
+SpacesBeforeTrailingComments: 1
+SortIncludes: false
```

Comment:
Be nice, no?

Reply:
Not sure how the sorting works, but in some contexts, being able to decide which file is included first is very important.

Reply:
I'd be in favor. 🙂

Reply:
Please don't change this without actually testing how it works. It is quite common to need the configure generated header to come first.